src/database/schemas.ts

```diff
@@ -18,7 +33,7 @@ const typeMap = {
 export default class Database {
   static async getObjects(schema: string, type: SQLType, details: PageData = {}): Promise<BasicSQLObject[]> {
     // The database doesn't store the object name with the quotes.
-    schema = Statement.noQuotes(Statement.delimName(schema));
+    schema = Statement.noQuotes(Statement.delimName(schema, true));
```

Comment:
Calls to getObjects are likely driven by the user navigating the Schema Browser.  In that context, the schema name is not user input.  Further, it should already be properly delimited, so it only needs to be stripped of its quotes.
```suggestion
    schema = Statement.noQuotes(schema);
```

Reply:
Thinking more about this, I guess maybe the code is necessary for a non-delimited schema name, as it displays in lowercase and needs to go through `delimitName` to get upper-cased.  If so, disregard this suggestion.